tests/profile/test_set_default_profile.py

```diff
@@ -0,0 +1,87 @@
+import logging
+import random
+
+from selenium.webdriver import Firefox
+from selenium.webdriver.common.by import By
+
+from modules.page_object import AboutProfiles
+
+
+def test_set_default_profile(driver: Firefox):
+    """
+    C130792, set the default profile through the firefox browser
+    """
+    about_profiles = AboutProfiles(driver).open()
+
+    # get the profiles container, extract all relevant children under it.
+    profile_container = about_profiles.get_element("profile-container")
+    profiles = profile_container.find_elements(By.XPATH, "./child::*")
+
+    # verify that some profile is the default
+    profile_header_in_use = about_profiles.get_element(
+        "profile-container-item-default-header"
+    )
+    assert profile_header_in_use is not None
+
+    cur_default = -1
+
+    # find index that is the current default
+    for i in range(len(profiles)):
+        logging.info(f"Currently searching row {i} for the default profile")
+        cur_profile = profiles[i]
+        table_rows = about_profiles.get_element(
+            "profile-container-item-table-row",
+            multiple=True,
+            parent_element=cur_profile,
+        )
+        first_row = table_rows[0]
+        # find the current default profile
+        default_profile_information = about_profiles.get_element(
+            "profile-container-item-table-row-value", parent_element=first_row
+        )
+        if default_profile_information.get_attribute("innerHTML") == "yes":
+            logging.info(f"Found the default profile at {i}!")
+            cur_default = i
+            break
+
+    # no default profile could be found
+    if cur_default == -1:
+        logging.warn("Could not find a currently active default profile.")
+        assert False
+
+    # select a non default profile randomly
+    profile_index = random.randint(0, len(profiles) - 1)
```

Comment:
could make a set of all the profiles that are not the current, then choose a random from that, would avoid iteration.

Reply:
Got it. Fix was just pushed